lambdas/receiver.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+
+// Set the region
+AWS.config.update({region: process.env.AWS_REGION});
```

Comment:
We don't have to do this on `process.js` because bucket is not tied to region, right?

Reply:
I'm not actually certain we need to do this, but the sample code I was adapting did.